DESCRIPTION

```diff
@@ -21,12 +21,19 @@ Suggests:
     httr,
     knitr,
     shinytest,
-    testthat (>= 2.0)
+    testthat (>= 2.0),
+    roxytypes,
+    roxylint
+Remotes:
+    dgkf/roxytypes,
```

Comment:
how about moving the repo to `openpharma` or `genentech` soon?

Reply:
Sure, no problem for me. In order to transfer it to `openpharma` I need my `@dgkf` account to be added to the org. My only concern would be that it is very general and independent of pharma, but I think otherwise it's the best option.

Reply:
yeah we could also move it later somewhere else (e.g. ropensci or other) once it is mature